module6/lessons/Week1/JobHuntHustle.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Job Hunt Hustle Intro
+---
+
+## Introduction
+
+Find job you're interested in, apply to job, interview for job, receive offer: that's the process right? Sometimes that works, but more often than not the path towards a job offer is more winding and involves more "hustle" on your end.
+
+What do I mean by "hustle"? I think of the hustle as putting in extra time and effort to help you stand out in comparison to the other candidates.
+
+There are so many different activities this can include, and by the end of today we will have a list of "Job Hunt Hustle" activities to choose from. 
+
+For "Job Hunt Hustle" we are not going to focus on things that help you ace the interview or grow your qualifications. We instead want to focus on things that help you move closer on the path towards jobs you want. Things that that help get your foot in the door and help you stay top of mind to potential employers.
+
+### Examples of Job Hunt Hustle
+
+#### Example 1
+[Thoughts from Ian Douglass, a former Turing Instructor](https://twitter.com/iandouglas736/status/1593731985142226944)
+
+#### Example 2
+[Thoughts from a LinkedIn Career Coach](https://www.linkedin.com/feed/update/urn:li:activity:7121159930262261760/)
+
+#### Example 3

```

Comment:
I think these examples 3 and 4 are helpful to give very concrete examples of when hustling has paid off, not just from someone out on the internet but from someone they know. And hopefully, others (maybe Isiah?) will share times they have husted and it paid off. But I'm not sure if these make sense from an outside perspective or if they are too long. Would love to hear your thoughts!

Reply:
I LOVE that you included these, and I totally agree that it will help them cement these ideas by seeing them in actual practice!!